### PR TITLE
Add C# to 2D Movement Overview

### PR DESCRIPTION
**Pull request by skyace65**
Adds C# code to the 2D  Movement Overview page. Closes issue #1292

### CODE REVIEW

File: tutorials/2d/2d_movement.rst [hunk 1]
```diff
@@ -60,6 +61,46 @@ Add a script to the kinematic body and add the following code:
...
+    using Godot;
+    using System;
+
+    public class Movement : KinematicBody2D
+    {
+        [Export] public int Speed = 200;
+
+        Vector2 velocity = new Vector2();
+
+        public void _GetInput()
```
Comment (mysticfall) [inline tutorials/2d/2d_movement.rst]: I think you forgot this, as we just discuss about it on Discord :)

Comment (skyace65) [reply tutorials/2d/2d_movement.rst]: Make `velocity` upper case?

Comment (mysticfall) [reply tutorials/2d/2d_movement.rst]: No, I meant `_Input`, sorry! It seems that you changed the name already in the following snippet but forgot to do it here.

**Comment by skyace65** [comment]
That should be fixed now.

**Pull request merged**